demo/datasource/source.py

```diff
@@ -24,10 +24,15 @@ def load_vocab(file_path):
 def reader(file_path, vocab):
     with open(file_path, "r") as reader:
         for line in reader:
+            split_line = line.split()
+            # ATIS every row starts/ends with BOS/EOS: remove them
+            # except for e.g. intents line, which have a single token
+            if len(split_line) > 1:
+                split_line = split_line[1:-1]
```

Comment:
This is only correct if the for this particular ATIS dataset but is breaking every other use case. This logic belongs to a Tensorizer and it's up to the user to use the correct one. Please remove this change.

Reply:
Hm, I thought this file was made specifically for the demo, which uses the ATIS dataset.
At either case, since the main fix is already implemented, and you'd like this removed, I'll close the PR.

Thanks!